framework_tvb/tvb/interfaces/web/static/js/spatial/stimulus.js

```diff
@@ -70,6 +70,7 @@ function _STIM_saveWeightForSelectedNodes() {
         weightElement.val("");
         _STIM_server_update_scaling();
     }
+    return newWeight;
```

Comment:
Indeed, the scaling on the stimulus was not shown when we clicked "Apply", only after a Refresh.